src/Open3270Library/CommFramework/Audit.cs

```diff
@@ -24,8 +24,10 @@
 using System;
 using System.Web;
 using System.Text;
-using System.Security; 
+using System.Security;
+#if NET35 || NET40
```

Comment:
Does https://www.nuget.org/packages/System.Security.Permissions/ not support this?

Reply:
At the time I submitted my original PR I hadn't looked into this. When I ran the portability analyzer it indicated that the CAS methods were not in .NET Standard 2.0, so I only included them in the .NET Framework builds.

I'm no expert on CAS as it relates to .NET Standard 2.0, but if you believe its worth exploring adding this package as a new dependency to keep the CAS method calls, we could.